### DISCUSSION TITLE
display a part multiple times on the same view in different zones

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#14255] [Q&A]
Good evening everyone,

Is it possible to display a content part multiple times in the same view, in a different way and in different zones?

Thank you

**Comment by ns8482e** [comment]
It depends on your needs. However one way is to utilize Wrapper. 

Every shape is rendered and saved to a property of wrapper `Model.Metadata.ChildContent`.

**Reply by PiemP** [reply]
I have to use the same information in different way in the same view: 
- I have to show the information in the header to give to user a status about this information
- I have to edit this information in a form 

All in the same page. I have `DisplayDriver` and if I add an extra call to `Location` method on the same display type but on a different Wrapper, can I do this?

**Reply by ns8482e** [reply]
I guess every time you call BuildDispay it will create new shape to render and you can call it different places.

If I were you, I would create builddisplay only once and render shape in JavaScript variable in wrapper shape and use dom manipulation to diplomat wherever on same page.

**Reply by PiemP** [reply]
I don't really understand how your solution works... probably I'm far to understand wrappers, shapes etc etc.

I have to better understand all the thigs that goes around shape ... is difficult to me have all the things clear in mind ... for now I have solved by using a Shape of a part that is related to the edit operation of the view: I have retrieved the information from the Model.
